torch/nested/_internal/nested_tensor.py

```diff
@@ -15,11 +17,19 @@ def get_tensor_symint(tensor, *, coeff=1):
     global _tensor_id_counter
     tensor_symint = _tensor_symint_registry.get(tensor)
     if tensor_symint is None:
-        tensor_symint = torch._C._get_nested_int(_tensor_id_counter, coeff)
-        _tensor_id_counter += 1
-        _tensor_symint_registry[tensor] = tensor_symint
-    return tensor_symint
+        if isinstance(tensor, FunctionalTensor):
+            tensor = torch._from_functional_tensor(tensor.elem)
+            return get_tensor_symint(tensor, coeff=coeff)
+        elif isinstance(tensor, FakeTensor):
+            shape_env = tensor.fake_mode.shape_env
+            tensor_symint = shape_env.create_unbacked_symint()
+            # Do we need to constrain as size?
```

Comment:
why would this path ever be hit if we manually update the registry for fake tensors during tensor_unflatten?